sklearn/metrics/_plot/confusion_matrix.py

```diff
@@ -155,7 +155,7 @@ def plot_confusion_matrix(estimator, X, y_true, sample_weight=None,
         Includes values in confusion matrix.
 
     normalize : {'true', 'pred', 'all'}, default=None
-        Normalizes confusion matrix over the true (rows), predicited (columns)
+        Normalizes confusion matrix over the true (rows), predicted (columns)
         conditions or all the population. If None, confusion matrix will not be
```

Comment:
same here lol

Reply:
I copied it from here :). Seeking a bit more, this is defined in this way on wikipedia: https://en.wikipedia.org/wiki/Confusion_matrix